src/flows/pipes/Notification/endpoints/HTTP/index.ts

```diff
@@ -72,9 +72,11 @@ export default register => {
         }, [])
         .join(', ')
 
-      return `http["endpoint"](url: "${data.url}")(mapFn: (r) => {
-      return {headers: {${_headers}}, data: json["encode"](v: "${TEST_NOTIFICATION}")}
-  })
+      return `http.post(
+        url: "${data.url}",
+        data: json["encode"](v: "${TEST_NOTIFICATION}"),
+        headers: {${_headers}}
+      )
```

Comment:
if we make this change here, we should also make it for line 48 in the file above, since the two are related, this being the `testable` version of that (that is to say, it's the one the user tests out by pressing the button, rather than by generating a task)

Reply:
good call. updated. Body payload also compared (see readme) --> no change.